src/sheet_metal_conus.py

```diff
@@ -21,6 +21,11 @@ def normalize(p1, p2):
     yDiff = p2[1] - p1[1]
     magn = calc_dist_between_points(p1,p2)
     return (xDiff/magn, yDiff/magn)
+def polar_to_cartesian(cx, cy, radius, angle):
+    " So we can make arcs in the 'A' svg syntax. "
+    angle_radians = math.radians(angle)
+    return (cx + (radius * math.cos(angle_radians)),
+            cy + (radius * math.sin(angle_radians)))
```

Comment:
To draw arcs using path - we need to do this calc instead of using eth sodipodi arc code as currently.